addons/nuxeo-template-rendering/nuxeo-template-rendering-core/src/main/java/org/nuxeo/template/serializer/executors/XMLSerializer.java

```diff
@@ -0,0 +1,207 @@
+package org.nuxeo.template.serializer.executors;
```

Comment:
Missing copyright. If this file (or another) is derived from code copied from a Nuxeo-copyrighted file, then please use the Nuxeo copyright.